docs: update adapter-node.md (#9309)
sveltejs/kit: documentation/docs/25-build-and-deploy/40-adapter-node.md

First, build your app with `npm run build`. This will create the production server in the output directory specified in the adapter options, defaulting to `build`.

> You will need the output directory, the project's `package.json`, and the production dependencies in `node_modules` to run the application. Production dependencies can be generated (after package-lock.json copied here too) with `npm ci --omit dev` (you can skip this step if your app doesn't have any dependencies). You can then start your app with this command:

Reviewer: two sets of parentheses in one sentence feels like a bit much. we should put the file name in backticks
```suggestion
You will need the output directory, the project's `package.json`, and the production dependencies in `node_modules` to run the application. Production dependencies can be generated by copying the `package.json` and `package-lock.json` and then running `npm ci --omit dev` (you can skip this step if your app doesn't have any dependencies). You can then start your app with this command:
```

```

> ORIGIN=https://my.site node build
> ORIGIN=http://localhost:3000 node build
> ORIGIN=http://127.0.0.1:4000 node build

Reviewer 2: ```suggestion
ORIGIN=https://my.site node build

# or e.g. for local previewing and testing
ORIGIN=http://localhost:3000 node build
```
